tests/test_click_through.py

```diff
@@ -0,0 +1,15 @@
+# from unittest import TestCase
+import unittest
+
+from hypothesis import given
+from hypothesis.strategies import text
+
+from browse.domain.clickthrough import create_hash, is_hash_valid
+
+
+class TestClickthrough(unittest.TestCase):
+
+    @given(text(), text())
```

Comment:
👍 It's nice to see property-based testing; hypothesis seems like a solid library for doing this!